### PR TITLE
Enhacement/use editor api

### PR DESCRIPTION
**Pull request by lagartoverde**
To be honest I don't know a lot of this plugin and it's all in dutch, as far as I know everything looks good and it works, but if you could test a little bit it would be great.
Things missing I'm aware of:
- Deleting doesn't trigger the plugin as the deleting doesn't trigger a content change (to be fixed on the editor)
- It only checks for being inside a besluit it doesn't check it's on motivering and not on cites (to be fixed on another PR)

### CODE REVIEW

**Comment by abeforgit** [comment]
Needs: 
- insertion logic is still using old api (no longer works)
- bug with text-matching not correctly limiting search range

File: addon/utils/legislation-types.js [hunk 1]
```diff
@@ -18,8 +18,6 @@ const LEGISLATION_TYPES = {
   wetboek: 'https://data.vlaanderen.be/id/concept/AardWetgeving/Wetboek',
   'gecoordineerde wetten':
     'https://data.vlaanderen.be/id/concept/AardWetgeving/GecoordineerdeWetten',
-  'gecoördineerde wetten':
```
Comment (nvdk) [inline addon/utils/legislation-types.js:21]: you can't just remove this, we still want to match it

Comment (lagartoverde) [reply addon/utils/legislation-types.js:21]: It's the one that was duplicated

**Comment by nvdk** [comment]
can I just say that the type in the branch name trips me up each time I look at this :)

**Comment by nvdk** [comment]
> 2\. odd behaviour when backspacing/adding a space in pasted content:

this is fixed by https://github.com/lblod/ember-rdfa-editor/pull/247

**Pull request merged**